devices/models.py

```diff
@@ -0,0 +1,79 @@
+from django.db import models
+
+from users.models import User
+
+
+class Building(models.Model):
+    name = models.CharField(max_length=100, null=True)
```

Comment:
Django wtf. Property named null??????????????????????????????

Reply:
it's just field, id is a PK